### PR TITLE
Math Magicians: tests

### PR DESCRIPTION
**Pull request by rabayarabu**
- Write unit tests with React Testing Library.
- Write unit tests with snapshots with Jest.

### CODE REVIEW

File: src/testing/operate.test.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  });
+  test('Performs division correctly ', () => {
+    const result = operate('12', '4', '÷');
+    expect(result).toEqual('3');
+  });
+
+  test("return Can't divide by 0.", () => {
+    const result = operate('60', '0', '÷');
+    expect(result).toEqual("Can't divide by 0.");
+  });
```
Comment (danmainah) [inline src/testing/operate.test.js:24]: - [ ] @rabayarabu and @DebyGrey great work adding the tests for the operate.js file, however, you forgot to add a test for the modulus operator, kindly consider adding a test for it.

File: src/testing/Home.test.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Home from '../components/Home';
+
+test('renders correctly', () => {
+  const tree = renderer.create(<Home />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```
Comment (danmainah) [inline src/testing/Home.test.js:8]: - [ ] @team great work adding test for your components, however you forgot to add test for the `Navbar` and `App` components , kindly consider adding test for them

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @rabayarabu and @DebyGrey , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  Create unit tests using jest for most components and operate and calculate modules.
✔️  All tests are passing.

# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/testing/Home.test.js (hunk 2 above)
Comment (peterrobert) [reply src/testing/Home.test.js:8]: This test has been done well

**Review by peterrobert — APPROVED** [review]
Hi @DebyGrey  and @rabayarabu , 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**